charter.md

```diff
@@ -0,0 +1,32 @@
+# Style Team
+
+## Aims: Evolving the Rust Style over time
+
+1. Selecting styling for new Rust constructs
+2. Evolving the existing style over the course of Rust editions (without breaking backwards compatibility)
+3. Defining mechanisms to evolve the Rust style while taking backwards compatibility into account, such as via Rust editions or similar mechanisms
+
+## Domains
+
+1. [Style Guide]
+2. [T-style team repo](https://github.com/rust-lang/fmt-rfcs/)
+3. [#T-style] and #T-style/private zulip streams
+4. `rust-lang` issues with the `T-style` or `I-style-nominated` labels
+
+## Membership
+
+* Caleb Cartwright ([@calebcartwright](https://github.com/calebcartwright))
+* Jane Losare-Lusby ([@yaahc](https://github.com/yaahc))
+* Josh Triplett ([@joshtriplett](https://github.com/joshtriplett))
+* Michael Goulet ([@compiler-errors](https://github.com/compiler-errors))
+
+The Rust style team shall have at least 3 members and at most 8. If the team has fewer than 3 members it shall seek new members as its primary focus.
+
+Members of the style team are nominated by existing members. All existing members of the team must affirmatively agree to the addition of a member, with zero objections; if there is any objection to a nomination, the new member will not be added. In addition, the team lead or another team member will check with the moderation team regarding any person nominated for membership, to provide an avenue for awareness of concerns or red flags.
+
+### Becoming a member of T-Style
+
+Contributors who believe they'd be a good fit for the style team are encouraged to request an invite to the team. The style team will consider all such requests as as a proposal to become a team member and provide clear feedback to the contributor if there are any objections so that any issues may be resolved and the contributor can apply again in the future. The style team will not consider having previously been rejected as a basis for future rejections.
```

Comment:
```suggestion
Contributors who believe they'd be a good fit for the style team are encouraged to request an invite to the team, by reaching out via Zulip private message to one or more team members. The style team will consider all such requests as as a proposal to become a team member and provide clear feedback to the contributor if there are any objections so that any issues may be resolved and the contributor can apply again in the future. The style team will not consider having previously been rejected as a basis for future rejections.
```